app/scripts/app-init.js

```diff
@@ -12,9 +12,11 @@ function tryImport(...fileNames) {
 
 function importAllScripts() {
   const startImportScriptsTime = Date.now();
-  // applyLavaMoat has been hard coded to "true" as
-  // tryImport('./runtime-cjs.js') is giving issue with XMLHttpRequest object which is not avaialble to service worker.
-  // we need to dynamically inject values of applyLavaMoat once this is fixed.
+  /*
+   * applyLavaMoat has been hard coded to "true" as
+   * tryImport('./runtime-cjs.js') is giving issue with XMLHttpRequest object which is not avaialble to service worker.
```

Comment:
issue is not actually in runtime-cjs but in web3

Reply:
As I am looking into fixing this issue, I found that we are on pretty old version of Web3.js 0.20.0. Latest version of Web3.js already has this issue fixed. Should I just upgrade Web3 rather than patching it. It will be major version upgrade.

Reply:
I am working on fixing Web3 using patch.